### ISSUE TITLE
Wrong order of arguments to loss function in template example for finetuning

### ISSUE THREAD

**Issue by ajinkya-deogade** [PyTorchLightning/pytorch-lightning#2147]
Hi,
I observed that the order for the loss function in the example for finetuning is switched. Instead of
`self.loss(y_true, y_logits)` it should be `self.loss(y_logits, y_true)`

Below is the exact location of the error:
https://github.com/PyTorchLightning/pytorch-lightning/blob/bd49b07fbba09b1e7d8851ee5a1ffce3d5925e9e/pl_examples/domain_templates/computer_vision_fine_tuning.py#L246

**Comment by rohitgr7** [comment]
It's correct I think.
https://github.com/PyTorchLightning/pytorch-lightning/blob/bd49b07fbba09b1e7d8851ee5a1ffce3d5925e9e/pl_examples/domain_templates/computer_vision_fine_tuning.py#L206-L207

https://github.com/PyTorchLightning/pytorch-lightning/blob/bd49b07fbba09b1e7d8851ee5a1ffce3d5925e9e/pl_examples/domain_templates/computer_vision_fine_tuning.py#L192

**Comment by ajinkya-deogade** [comment]
Yes. I was used to seeing it in the (logits, target) order. Maybe we can change the loss to take in (self, logits, labels) in order to be consistent PyTorch conventions.

**Comment by Borda** [comment]
@ajinkya-deogade mind send a PR?
